keyboards/glencreag/wk84/readme.md

```diff
@@ -0,0 +1,25 @@
+# WK84 
+
+![WK84](https://i.imgur.com/Iau74ZK.png)
+
+
+A keyboard, Equipped with the WestBerry Q95 microcontroller.
+
+* Keyboard Maintainer: [GSKY](https://github.com/gksygithub)
+* Hardware Supported: WK84
```

Comment:
Please provide a link to where this keyboard can be purchased, or at least where more information can be found.
```suggestion
* Hardware Supported: WK84
* Hardware Availability: *Links to where you can find this hardware*
```